src/PresentationalComponents/NumberData/NumberData.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import '../../../src/PresentationalComponents/NumberData/NumberData.scss';
+
+export const NumberData = ({ data, dataSize }) => {
+    return (
+        <React.Fragment>
```

Comment:
I don't think you should need a fragment here since the `p` is the wrapper below.